Project.toml

```diff
@@ -18,7 +18,7 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 AxisKeys = "0.1.18, 0.2"
-DynamicPPL = "0.21, 0.22"
+DynamicPPL = "0.21, 0.24"
```

Comment:
Is there a reason for why you removed support for 0.22 but kept 0.21? And why only 0.24 but not 0.23 was added?

Reply:
No reason besides fast editing and shallow understanding of the compat syntax :sweat_smile: